### PR TITLE
Introduce govuk-components gem and first component

### PR DESCRIPTION
**Pull request by cesidio**

### CODE REVIEW

**Review by david-mears-dfe — COMMENTED** [review]
The resulting HTML appears to contain some duplication: there are two divs with the classes "govuk-panel govuk-panel--confirmation"

```html
<div class="govuk-panel tvs-panel govuk-panel--confirmation">
  <span class="hod-checkmark">
    <svg focusable="false" height="40" role="presentation" viewBox="0 0 25 25" width="40" xmlns="http://www.w3.org/2000/svg">
    <path d="M25,6.2L8.7,23.2L0,14.1l4-4.2l4.7,4.9L21,2L25,6.2z" fill="currentColor"></path>
    </svg>
  </span>

  <div class="govuk-panel govuk-panel--confirmation">
    <h1 class="govuk-panel__title">
      Your email subscription has started
    </h1>

    <div class="govuk-panel__body">
      We sent an email confirmation to test@example.com.
    </div>
  </div>

</div>
```

**Comment by david-mears-dfe** [comment]
Could you check the text size with UX as it looks oddly big to me.

Screenshot 2020-11-23 at 13 11 28
Screenshot 2020-11-23 at 13 12 13

File: config/locales/en.yml [hunk 1]
```diff
@@ -497,8 +497,8 @@ en:
     new:
       title: Create your job alert
     unsubscribe:
-      confirmation: You will no longer receive updates for this alert.
-      header: You have been unsubscribed from this email
+      confirmation: You will no longer receive updates for this job alert.
+      header: You have been unsubscribed from this alert
```
Comment (david-mears-dfe) [inline config/locales/en.yml:501]: Let's change this text either in both places, or in neither (see above screenshot of creating a subscription for NQTs/creating one normally). Also, please run it past Content Design.

**Review by alexbowen — APPROVED** [review]

**Comment by alexbowen** [comment]
actually im not sure this is right, lets catch up on a call first thing in morning. theres a few things to discuss and align on here

**Comment by alexbowen** [comment]
> The resulting HTML appears to contain some duplication: there are two divs with the classes "govuk-panel govuk-panel--confirmation"
> 
> ```
> <div class="govuk-panel tvs-panel govuk-panel--confirmation">
>   <span class="hod-checkmark">
>     <svg focusable="false" height="40" role="presentation" viewBox="0 0 25 25" width="40" xmlns="http://www.w3.org/2000/svg">
>     <path d="M25,6.2L8.7,23.2L0,14.1l4-4.2l4.7,4.9L21,2L25,6.2z" fill="currentColor"></path>
>     </svg>
>   </span>
> 
>   <div class="govuk-panel govuk-panel--confirmation">
>     <h1 class="govuk-panel__title">
>       Your email subscription has started
>     </h1>
> 
>     <div class="govuk-panel__body">
>       We sent an email confirmation to test@example.com.
>     </div>
>   </div>
> 
> </div>
> ```

i know whats going on here, need to talk to @cesi on a call really

**Review by alexbowen — APPROVED** [review]
all good pending the SVG PR getting merged

**Review by cpjmcquillan — APPROVED** [review]

**Pull request merged**